src/partials-js/chart.js

```diff
@@ -1,152 +1,274 @@
 'use strict';
+import axios from 'axios';
 import Chart from 'chart.js/auto';
+import format from 'date-fns/format';
+import { coordinates } from './data_fetch';
 //API Key
-const key = '86882c431a5c1fa03f48939e3b313043&units=metric';
+const apiKey = '86882c431a5c1fa03f48939e3b313043&units=metric';
+// https://api.openweathermap.org/data/2.5/forecast/daily?q=London&cnt=3&appid=86882c431a5c1fa03f48939e3b313043&units=metric
 // https://api.openweathermap.org/data/2.5/forecast?lat=57&lon=-2.15&cnt=5&appid=86882c431a5c1fa03f48939e3b313043&units=metric
+// https://api.openweathermap.org/data/2.5/onecall?lat=57&lon=-2.15&exclude=daily&appid=86882c431a5c1fa03f48939e3b313043&units=metric
 //DOM imports
 const ctx = document.getElementById('myChart');
-//Test Array
-const testArray = {
-  cod: '200',
-  message: 0,
-  cnt: 5,
-  list: [
+//Axios get function
+async function getDailyWeather(coordinates, apiKey) {
+  await axios
+    .get(
+      `https://api.openweathermap.org/data/2.5/onecall?lat=${coordinates.lat}&lon=${coordinates.lon}&exclude=hourly,minutely&appid=${apiKey}`,
+    )
+    .then(Response => {
+      apiResponse = Response.data.daily;
+      return apiResponse;
+    })
+    .catch(error => {
+      return error;
+    });
+}
+//Api response
+const apiResponse = {
```

Comment:
to nadal ma być wypełniona zmianna ? jak rozumiem to było do testów

Reply:
Tak bo tak jak w głównej zmiennej nie mogę załadować wyszukania z API przez ten stos zadań